wazp/callbacks/dashboard.py

```diff
@@ -245,27 +253,49 @@ def create_buttons_and_message() -> html.Div:
         is_open=False,
         color="success",  # warning or danger
         style={
-            "margin-right": "10px",
+            "margin-right": "150px",
             "margin-left": "10px",
             "margin-top": "5px",
         },
     )
 
+    clipboard = dcc.Clipboard(
+        id="clipboard",
+        title="Copy full path to clipboard",  # tooltip text
+        n_clicks=0,
+        style={
+            "display": "inline",  # visible
+            "fontSize": 20,
+            "margin-right": "10px",
+            "margin-left": "10px",
+        },
+    )
+
+    clipboard_wrapper = html.Div(
+        id="clipboard-wrapper",
+        children=[
+            clipboard,
+        ],
+        style={"display": "none"},  # hidden
```

Comment:
Is there any way to have the clipboard button _on_ the export message? Which I think is the `dbc.Alert` above, right? The green popup box thingy.
(Apologies for bike-shedding)

Reply:
yes! it turns out it's way easier than what I was trying a few days ago... 🤩